bib/internal/setup/setup.go

```diff
@@ -100,23 +102,43 @@ func Validate() error {
 	return nil
 }
 
-// ValidateHasContainerStorageMounted checks that the container storage
-// is mounted inside the container
-func ValidateHasContainerStorageMounted() error {
+var insideContainer = func() (bool, error) {
 	if err := exec.Command("systemd-detect-virt", "-c").Run(); err != nil {
 		if exitErr, ok := err.(*exec.ExitError); ok {
 			// not running in a container, just exit
 			if exitErr.ExitCode() == 1 {
-				return nil
+				return true, nil
```

Comment:
`systemd-detect-virt -c` exits with 1 if it is NOT running in a container. Thus, I think that this function should return false here (because it's called `insideContainer`). IoW the return values are as far as I understand flipped.

Reply:
Thanks, I'm a Muppet. Thanks for catching it! I simplified and reuploaded.